src/__tests__/model.test.ts

```diff
@@ -60,7 +60,7 @@ describe('model', () => {
 
   beforeEach(() => {
     doc = {
-      _id: new ObjectId(),
+      _id: new ObjectId('62b3876eb20d64c7bb577874'),
```

Comment:
Why is this ID needed here?

Reply:
You're right, not needed anymore, I wasn't using the `result` variable to test the input parameters at first and was hard coding the expected input.